### ISSUE TITLE
IPLD Use Cases and Call for Demos

### ISSUE THREAD

**Issue by RichardLitt** [ipld/ipld#14]
IPLD is implemented, and has been deployed for a while, in both js-ipfs and go-ipfs. It's there - we haven't made an announcement about it, though. We are working on [the website](https://github.com/ipld/website). Now, we need some **good demos** to nail down the interface for operating over IPLD 

There is a lot of work to be done with regards to interface design, too. See https://github.com/ipfs/interface-ipfs-core/pull/99 and https://github.com/ipfs/interface-ipfs-core/issues.

@jonnycrunch: You mentioned you're interested in exploring use-cases for identity management with W3C and cross-over with JSON-LD/Verifible Claims Task Force. Do you think that would be a good demo for IPLD?  

Ideas: Build some demos, and PR them here or in ipld/examples or ipld/demos. Another idea would be to have a blog post. 

If you know any good demos, discuss below. Or if you know anyone who would be interested in this thread, direct them here.

**Comment by jonnycrunch** [comment]
@RichardLitt : yes, I do.  I have been exploring JSON-LD for self-sovereign digital identity solutions, but lately have been contemplating IPLD as the '@context' link would need to be a persistent URL ( perfect application for IPFS).  in JSON-LD the '@context' attribute aids in semantic interoperability of the data.  The same could be done in IPLD and I'd love to explore this concept.  The issue with JSON-LD using IPFS for the context is the JSON content for the '@context' needs to be deterministic.  I'll join the IPLD call tomorrow to discuss.

**Comment by jonnycrunch** [comment]
@RichardLitt @flyingzumwalt  I have an update use case that I worked on for the RebootingWebofTrust meeting last week in Boston. 

I implemented the DID ( Decentralized Identifiers) on top of IPFS ( more specifically IPNS). I called it IPID ( for interplanetary identifiers). 

checkout:  https://www.github.com/jonnycrunch/ipid   

However, I'd like to resolve a IPID method specific identifier, like did:ipid:QmbFuwbp7yFDTMX6t8HGcEiy3iHhfvng89A19naCYGKEBj to a DID document using IPLD. but I need to figure our how to resolved IPLD resources in IPNS.  Any thoughts? 

@jbenet I'll be in Mountain View next week if you'd like to meet up to discuss.

**Comment by diasdavid** [comment]
@b5 just showed today a really awesome demo of using IPFS to distribute datasets and run queries over them, a great usecase for IPLD Selectors and Transformations.

I'll post the url to the demo here once I finish uploading it!

**Comment by b5** [comment]
Super fired up to discuss this! We've long intended to implement IPLD, relevant section of our white paper: http://qri.io/papers/deterministic_querying/#integration-with-linked-data-and-linked-open-data

**Comment by b5** [comment]
@diasdavid mentioned scheduling a call to discuss. Not sure what time scale you were thinking David, so how's next week sound? If an availability poll helps, here's one: https://www.when2meet.com/?6507620-atNhM

@jonnycrunch from reading above sounds like you're also fired up about JSON-LD & IPLD intersecting, I'd love to hear more of your thinking on the subject while we're on the subject

**Comment by diasdavid** [comment]
This is a neat tool! Thanks @b5. 

I only realized after that times were in EST and so I added a wrong set of available hours. Seems that you are free at 11:30am this Friday https://www.worldtimebuddy.com/?qm=1&lid=5,2267057&h=5&date=2017-12-1&sln=11.5-12.5. Would that work for you too, @jonnycrunch ?

**Comment by b5** [comment]
Hey @diasdavid, why don't we go ahead & schedule for 11:30am EST this Friday? I'll set aside some time to brush up on IPLD & DEX beforehand!

**Comment by jonnycrunch** [comment]
@diasdavid good for me.

**Comment by diasdavid** [comment]
Ah snap! I didn't block this on my schedule right away and got double booked. Would an hour later work for you as well?

**Comment by b5** [comment]
Sure! I can make 12:30EST work as well.

**Comment by diasdavid** [comment]
Sweet! @nicola @Stebalien you might want to join this one as well :)

## Zoom info:

```
Hi there, 

David Dias is inviting you to a scheduled Zoom meeting. 

Join from PC, Mac, Linux, iOS or Android: https://protocol.zoom.us/j/123535118

Or iPhone one-tap :
    US: +16468769923,,123535118#  or +16699006833,,123535118# 
Or Telephone:
    Dial(for higher quality, dial a number based on your current location)：
        US: +1 646 876 9923  or +1 669 900 6833  or +1 408 638 0968 
    Meeting ID: 123 535 118
    International numbers available: https://protocol.zoom.us/zoomconference?m=75A8oPS2FI_IZH_nOoMu3XddFHDocjBy


```

**Comment by jonnycrunch** [comment]
yep, ok with me
